### PR TITLE
cbu updater inst steps

### PR DESCRIPTION
**Pull request by brett-johnson**

### CODE REVIEW

File: content/cloud-backup/update-or-install-the-cloud-backup-agent-on-linux.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ variable with the API key:
 
     echo -n 'API key: ' && read -s apikey && export apikey && echo ''
 
-To install the updater where an agent is already registered, simply run the ./cloudbackup-updater.
-To both install the updater and install and register the Cloud Backup agent, run
-the following command:
+Change to the extracted folder. To install the updater where an agent is already
+registered, simply run the ./cloudbackup-updater. To both install the updater and install
```
Comment (catlook) [inline content/cloud-backup/update-or-install-the-cloud-backup-agent-on-linux.md:45]: ... the `./cloudbackup-updater`. To both ...

**Review by catlook — CHANGES_REQUESTED** [review]
put the command in back ticks to format it.

**Review by catlook — APPROVED** [review]
good to go

**Review by Mint-6263 — COMMENTED** [review]
Ok

**Pull request merged**